src/app/commands/cmd_save_file.cpp

```diff
@@ -112,9 +112,12 @@ void SaveFileBaseCommand::onLoadParams(const Params& params)
     m_adjustFramesByTag = false;
   }
 
-  std::string useUI = params.get("useUI");
+  const std::string useUI = params.get("useUI");
   m_useUI = (useUI.empty() || (useUI == "true"));
 
+  const std::string addToRecentFiles = params.get("addToRecentFiles");
+  m_addToRecentFiles = (addToRecentFiles == "true");
```

Comment:
I think I would prefer a shorter param name like "addToRecent", and check if it's empty (in that case we add to recent, as it's the default behavior):
```suggestion
  const std::string addToRecent = params.get("addToRecent");
  m_addToRecent = (addToRecent.empty() || (addToRecent == "true"));
```
In a future we have to change the parameters of all commands to NewParams, but that is a refactor for the future.